AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1036,6 +1037,21 @@ - (void)__setNeedsLayout
   }
 }
 
+// If not rasterized (and therefore we certainly have a view or layer),
+// Send the message to the view/layer first, as scheduleNodeForDisplay may call -displayIfNeeded.
+// Wrapped / synchronous nodes created with initWithView/LayerBlock: do not need scheduleNodeForDisplay,
+// as they don't need to display in the working range at all - since at all times onscreen, one
+// -setNeedsDisplay to the CALayer will result in a synchronous display in the next frame.
+- (void)__setNeedsDisplay
+{
+  BOOL nowDisplay = ASInterfaceStateIncludesDisplay(_interfaceState);
+  // FIXME: This should not need to recursively display, so create a non-recursive variant.
```

Comment:
This is an important follow-up. I know this has existed for a couple months now, but could you add it to a list of follow-ups? I am probably the best person to deal with this, but it would be helpful to have the direct reminder as we talk about next steps following this.  a github issue would just get lost :)